adapters/xyfinance/src/index.ts

```diff
@@ -0,0 +1,104 @@
+import { getEvents } from "./sdk/subgraphDetails";
+
+export const BATCH_SIZE = 1000;
+export const ETH_VAULT_ADDRESS = "0xa5cb30e5d30a9843b6481ffd8d8d35dded3a3251";
+export const ETH_VAULT_DEPLOY_BLOCK = 240;
+export const USDC_VAULT_ADDRESS = "0x9d90cfa17f3afcee2505b3e9d75113e6f5c9e843";
+export const USDC_VAULT_DEPLOY_BLOCK = 206280;
+export const VAULT_TO_TOKEN: Record<string, string> = {
```

Comment:
@jaredWu0805  currently pools are hardcoded here. if there are new pools added, please submit a new pr to have them indexed